src/zoid/buttons/component.jsx

```diff
@@ -508,6 +507,12 @@ export const getButtonsComponent : () => ButtonsComponent = memoize(() => {
                 type:       'boolean',
                 value:      isSupportedNativeBrowser,
                 queryParam: true
+            },
+            
+            userExperienceFlow: {
+                type:       'string',
+                required:   false,
+                value:      getUserExperienceFlow
             }
```

Comment:
Does this need `queryParam: true` so the param is persisted in the querystring?

Reply:
I think it's only needed on the client right now

Reply:
Ok cool. Never mind then.